bin/docker-run.sh

```
@@ -6,8 +6,11 @@ export PATH="$HOME/bin:$HOME/vendor/bundle/bin:$HOME/vendor/bundle/ruby/3.2.3/bi
 
 rm -rf tmp/pids
 
-# Start Sidekiq in the background
-bundle exec sidekiq -C config/sidekiq.yml &
+# Run migrations before starting services
+bundle exec rake cf:run_migrations db:migrate || echo "Warning: Migrations failed, continuing anyway..."
+
+# Start Sidekiq in the background (don't fail the container if it can't start)
+(bundle exec sidekiq -C config/sidekiq.yml || echo "Warning: Sidekiq failed to start, continuing with Puma only...") &
```

Is this just for debugging or do we want these to fail silently? If migrations dont run the app probably wont boot anyway so we should probably just let it fail here